models/api_host.go

```diff
@@ -50,10 +49,7 @@ func (h *Host) ConvertNMHostToAPI() *ApiHost {
 	}
 	a.IsStatic = h.IsStatic
 	a.ListenPort = h.ListenPort
-	a.LocalRange = h.LocalRange.String()
-	if isEmptyAddr(a.LocalRange) {
-		a.LocalRange = ""
-	}
+	a.LocalListenPort = h.LocalListenPort
```

Comment:
what is the purpose of `LocalListenPort`?

Reply:
Probably no longer required but out of scope for this PR

Reply:
it was actually removed in PR #1949 , probably you need to rebase your branch with develop

Reply:
removed